pandas/tests/tseries/test_ticks.py

```diff
@@ -0,0 +1,207 @@
+# -*- coding: utf-8 -*-
+"""
+Tests for offsets.Tick and subclasses
+"""
+from datetime import datetime, timedelta
+
+import pytest
+import numpy as np
+
+from pandas import Timedelta, Timestamp
+from pandas.tseries import offsets
+from pandas.tseries.offsets import Hour, Minute, Second, Milli, Micro, Nano
+
+
+# ---------------------------------------------------------------------
+# Test Helpers
+
+tick_classes = [Hour, Minute, Second, Milli, Micro, Nano]
+
+
+def assertEq(offset, base, expected):
+    actual = offset + base
+    actual_swapped = base + offset
+    actual_apply = offset.apply(base)
+    try:
+        assert actual == expected
+        assert actual_swapped == expected
+        assert actual_apply == expected
+    except AssertionError:
+        raise AssertionError("\nExpected: %s\nActual: %s\nFor Offset: %s)"
+                             "\nAt Date: %s" %
+                             (expected, actual, offset, base))
+
+# ---------------------------------------------------------------------
```

Comment:
these lines are fine, put a comment (note that I find this much more readable than classes).